File: src/main/webapp/resources/js/admin/siteSettingService.js
app.service("siteSettingService", function ($http) {
    this.save = function (_data) {
        var req = {
            method: 'POST',
            url:   '/admin/siteSetting/edit',
            headers: {
                'Content-Type': 'application/json'
            },
            data: _data
        };
        return $http(req);
    };
    this.find = function () {
        var req = {
            method: 'POST',
            url:   '/admin/siteSetting/find',
            headers: {
                'Content-Type': 'application/json'
            }
        };
        return $http(req);
    }
});

File: src/main/java/com/hu/hy/service/SiteSettingService.java
package com.hu.hy.service;

import com.hu.hy.domain.SiteSetting;

/**
 * 类的功能，目的，描述等写在此处
 *
 * @author 胡玉波
 * @version 1.0
 * @(#) SiteSettingService.java 2017/03/16 21:52
 */
public interface SiteSettingService {

    SiteSetting edit(SiteSetting siteSetting);

    SiteSetting find();
}